src/PowerShellEditorServices.Protocol/Server/LanguageServerSettings.cs

```diff
@@ -249,58 +249,62 @@ public CodeFormattingSettings(CodeFormattingSettings codeFormattingSettings)
+                    { "IgnoreOneLineBlock", IgnoreOneLineBlock }
                 }},
-                {"PSUseConsistentIndentation", new Hashtable {
-                    {"Enable", true},
-                    {"IndentationSize", tabSize},
-                    {"PipelineIndentation", PipelineIndentationStyle },
-                    {"Kind", insertSpaces ? "space" : "tab"}
+                { "PSUseConsistentIndentation", new Hashtable {
+                    { "Enable", true },
+                    { "IndentationSize", tabSize },
+                    { "PipelineIndentation", PipelineIndentationStyle },
+                    { "Kind", insertSpaces ? "space" : "tab" }
                 }},
-                {"PSUseConsistentWhitespace", new Hashtable {
-                    {"Enable", true},
-                    {"CheckOpenBrace", WhitespaceBeforeOpenBrace},
-                    {"CheckOpenParen", WhitespaceBeforeOpenParen},
-                    {"CheckOperator", WhitespaceAroundOperator},
-                    {"CheckSeparator", WhitespaceAfterSeparator}
+                { "PSUseConsistentWhitespace", new Hashtable {
+                    { "Enable", true },
+                    { "CheckOpenBrace", WhitespaceBeforeOpenBrace },
+                    { "CheckOpenParen", WhitespaceBeforeOpenParen },
+                    { "CheckOperator", WhitespaceAroundOperator },
+                    { "CheckSeparator", WhitespaceAfterSeparator },
+                    { "CheckInnerBrace", WhitespaceInsideBrace },
+                    { "CheckPipe", WhitespaceAroundPipe },
                 }},
-                {"PSAlignAssignmentStatement", new Hashtable {
-                    {"Enable", true},
-                    {"CheckHashtable", AlignPropertyValuePairs}
+                { "PSAlignAssignmentStatement", new Hashtable {
+                    { "Enable", true },
+                    { "CheckHashtable", AlignPropertyValuePairs }
                 }},
-                {"PSUseCorrectCasing", new Hashtable {
-                    {"Enable", UseCorrectCasing}
+                { "PSUseCorrectCasing", new Hashtable {
+                    { "Enable", UseCorrectCasing }
                 }},
             };
 
             if (AutoCorrectAliases)
             {
+                // Empty hashtable required to activate the rule,
+                // since PSAvoidUsingCmdletAliases inherits from IScriptRule and not ConfigurableRule
                 ruleConfigurations.Add("PSAvoidUsingCmdletAliases", new Hashtable());
             }
 
-            return new Hashtable
+            return new Hashtable()
             {
-                {"IncludeRules", new string[] {
-                     "PSPlaceCloseBrace",
-                     "PSPlaceOpenBrace",
-                     "PSUseConsistentWhitespace",
-                     "PSUseConsistentIndentation",
-                     "PSAlignAssignmentStatement",
-                     "PSAvoidUsingCmdletAliases",
```

Comment:
This entry is actually not necessary and was removed in PR #1056, hence why this changed is brought back as well as the essential required thing is `ruleConfigurations.Add("PSAvoidUsingCmdletAliases", new Hashtable());` a few lines above